nucypher/blockchain/eth/deployers.py

```diff
@@ -1060,7 +1066,7 @@ def check_deployment_readiness(self, *args, **kwargs) -> Tuple[bool, list]:
         ]
         return super().check_deployment_readiness(additional_rules=adjudicator_deployment_rules, *args, **kwargs)
 
-    def _deploy_essential(self, contract_version: str, gas_limit: int = None, **overrides):
+    def _deploy_essential(self, contract_version: str, gas_limit: int = None, confirmations: int = 0, **overrides):
```

Comment:
Do you regard the confirmations of deployment distance from that of other operations?

Reply:
I prefer to deploy with at least 2 or 3 confirmations, but I don't think this PR should be opinionated on this matter.